lms/templates/student_account/hinted_login.underscore

```diff
@@ -0,0 +1,24 @@
+<div class="hint-wrapper">
+    <div class="section-title lines">
+        <h2>
+            <span class="text"><%- gettext("Sign in") %></span>
+        </h2>
+    </div>
+
+    <p><%- _.sprintf( gettext("Would you like to sign in using your %(providerName)s credentials?"), { providerName: hintedProvider.name } ) %></p>
+
+    <button class="action action-primary action-update proceed-button button-<%- hintedProvider.id %> hinted-login-<%- hintedProvider.id %>">
+        <div class="icon fa <%- hintedProvider.iconClass %>" aria-hidden="true"></div>
```

Comment:
Gotcha! Its tough keeping with your and your twin you've tandemly got working behind the scenes on both of these PRs :P

Thanks!